### PR TITLE
Features/rhpm ql05

### PR DESCRIPTION
**Pull request by bharathkeerthi**
features/RHPM_QL05 qualification duplicate check

### CODE REVIEW

File: web/src/app/admin/qualifications/qualifications.component.html [hunk 1]
```diff
@@ -15,7 +15,8 @@
                                     </div>
                                     <div class="col-md-9 col-sm-9 col-xs-12">
                                         <input autofocus type="text" class="full-input" id="qualifications" id="name" name="qualificationName" [(ngModel)]="qualificationsModel.name" required #qualificationName="ngModel"
-                                            minlength="3" maxlength="20">
+                                            minlength="3" maxlength="20" (keyup)="checkNameExits()">
+                                            <div *ngIf="isQualificationExists"  id="nameExists"  class="text">{{ 'QUALIFICATIONS.MESSAGE' | translate }}</div>
                                         <div *ngIf="qualificationName.errors && (qualificationName.dirty || submitted)" [hidden]="!qualificationName.errors.required" class="validation-error">
                                             {{ 'USER.REQUIRED' | translate }}
```
Comment (salilsaini868) [inline web/src/app/admin/qualifications/qualifications.component.html]: 'USER.REQUIRED' is it for user or qualification

File: web/src/app/admin/qualifications/qualifications.component.ts [hunk 2]
```diff
@@ -12,11 +12,10 @@ export class QualificationsComponent implements OnInit {
   qualifications: QualificationViewModel[] = [] as QualificationViewModel[];
   isCreateOrUpdate  = true;
```
Comment (salilsaini868) [inline web/src/app/admin/qualifications/qualifications.component.ts]: what is the purpose of this variable ?

File: web/src/app/admin/qualifications/qualifications.component.ts [hunk 3]
```diff
@@ -42,7 +41,19 @@ export class QualificationsComponent implements OnInit {
       }
     }
   }
-
+  checkNameExits() {
```
Comment (salilsaini868) [inline web/src/app/admin/qualifications/qualifications.component.ts]: change function  name to checkQualificationExists

**Review by salilsaini868 — CHANGES_REQUESTED** [review]

File: web/src/app/admin/qualifications/qualifications.component.ts (hunk 3 above)
Comment (bharathkeerthi) [reply web/src/app/admin/qualifications/qualifications.component.ts]: done

File: web/src/app/admin/qualifications/qualifications.component.html (hunk 1 above)
Comment (bharathkeerthi) [reply web/src/app/admin/qualifications/qualifications.component.html]: done

File: web/src/app/admin/qualifications/qualifications.component.ts (hunk 2 above)
Comment (bharathkeerthi) [reply web/src/app/admin/qualifications/qualifications.component.ts]: its required for edit field

Comment (salilsaini868) [reply web/src/app/admin/qualifications/qualifications.component.ts]: remove it and find out some another way.

**Review by salilsaini868 — APPROVED** [review]

**Pull request merged**